src/moonlink/src/storage/iceberg/rest_catalog_test.rs

```
@@ -103,3 +110,100 @@ async fn test_load_table() {
+        .await
+        .expect("Catalog creation fail");
+
+    let table_ident = guard.table.clone().unwrap();
+    let millis = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
+        .as_millis();
+
+    let mut table_updates = vec![];
+    table_updates.append(&mut vec![
+        TableUpdate::AddSnapshot {
+            snapshot: iceberg::spec::Snapshot::builder()
+                .with_snapshot_id(1)
+                .with_sequence_number(1)
+                .with_timestamp_ms(millis as i64)
+                .with_schema_id(0)
+                .with_manifest_list(format!(
+                    "s3://{}/{}/snap-8161620281254644995-0-01966b87-6e93-7bc1-9e12-f1980d9737d3.avro",
+                    NamespaceIdent::new(namespace.clone()).to_url_string(),
+                    table
+                ))
+                .with_parent_snapshot_id(None)
+                .with_summary(iceberg::spec::Summary {
+                    operation: iceberg::spec::Operation::Append,
+                    additional_properties: HashMap::new(),
+                })
+                .build(),
+        },
+        TableUpdate::SetSnapshotRef {
+            ref_name: MAIN_BRANCH.to_string(),
+            reference: SnapshotReference {
+                snapshot_id: 1,
+                retention: SnapshotRetention::Branch {
+                    min_snapshots_to_keep: None,
+                    max_snapshot_age_ms: None,
+                    max_ref_age_ms: None,
+                },
+            },
+        },
+    ]);
+
+    let table_commit_proxy = TableCommitProxy {
+        ident: table_ident.clone(),
+        requirements: vec![],
+        updates: table_updates,
+    };
+    let table_commit = table_commit_proxy.take_as_table_commit();
+
+    let table_obj = catalog
+        .update_table(table_commit)
+        .await
+        .unwrap_or_else(|_| panic!("Update table fail, namespace={namespace} table={table}"));
```

This is bad, since you discard the error propagated; why not `unwrap`?